drv8838/drv8838.go

```diff
@@ -0,0 +1,74 @@
+// Package drv8838 provides a driver to the DRV8838 Low-Voltage H-Bridge Driver
+// typically used to control DC motors.
+//
+// Datasheet: https://www.ti.com/lit/ds/symlink/drv8838.pdf
+//
+package drv8838 // import "tinygo.org/x/drivers/drv8838"
+
+import (
+	"machine"
+)
+
+const (
+	DIRECTION_FORWARD  = false
+	DIRECTION_BACKWARD = true
+)
+
+// Device is a motor with direction (phase), speed (enable) and sleep controlled by a DRV8838
+type Device struct {
+	enable machine.PWM
+	phase  machine.Pin
+	sleep  machine.Pin // Optional
+}
+
+// Device returns a new motor device
```

Comment:
```suggestion
// NewDevice returns a new motor device
```
Fixed name in comment of function